src/routes/show.svelte

```diff
@@ -84,23 +84,17 @@
   unsubscribe.application = application.subscribe(({
     repo_branch: repoBranch,
     repo_url: repoUrl,
-    env_variables: envVariables,
   }) => {
     if (repoBranch && repoUrl) {
       const queryParam = {
-        repoUrl,
-        repoBranch,
-        env_vars: envVariables
-          ? toQueryString(envVariables, ',')
-          : null,
```

Comment:
Is it correct we remove the `env_vars`?

Reply:
yes, as `env_vars` are part of the spec they're not being saved in the datasets

Reply:
uhm, no I mean we need to send the `env_vars` somehow to the deployment url right? Now we are not sending any `env_var` to the deployment url.